### PR TITLE
GPII-3867 - Update load data section

### PR DESCRIPTION
**Pull request by amatas**
Update the load of the data section in the README

[Rendered version](https://github.com/gpii-ops/gpii-infra/blob/f00b77119210efde4f9dabca8c5688b6492378c3/gcp/README.md#load-the-data-into-a-cluster)

### CODE REVIEW

File: gcp/README.md [hunk 1]
```diff
@@ -630,16 +630,13 @@ Note that we assume that you are going to perform these steps into an already up
...
-The port forwarding will be there until you hit _Ctrl-C_, so leave it running until we are done loading the preferences sets.
+1. Set up the env you are going to deal with: `rake sh`
 __Note__ that if you are going to perform this in production (prd) you should do it from the _prd_ folder and remember to use the _RAKE_REALLY_RUN_IN_PRD=true_ variable when issuing the commands against the production cluster.
+1. Open a port forwarding between the cluster's couchdb host:port and your local machine: `kubectl --namespace gpii port-forward couchdb-couchdb-0 5984 &`
+1. Get the current credentials for connecting to the couchdb from a flowmanager pod: `eval $(kubectl -n gpii exec flowmanager-[POD_ID] env | grep ^GPII_DATASOURCE_HOSTNAME | cut -f1 -d@)`
 
 Let's load the data, go back to the folder _testData/myDbData_ and run:
-1. `curl -d @gpiiKeys.json -H "Content-type: application/json" -X POST http://localhost:5984/gpii/_bulk_docs`
-1. `curl -d @.json -H "Content-type: application/json" -X POST http://localhost:5984/gpii/_bulk_docs`
+1. `curl -d @gpiiKeys.json -H "Content-type: application/json" -X POST $GPII_DATASOURCE_HOSTNAME@localhost:5984/gpii/_bulk_docs`
```
Comment (natarajaya) [inline gcp/README.md]: When running inside `rake sh`, secret variables can be used directly, i.e. `http://$TF_VAR_secret_couchdb_admin_username:$TF_VAR_secret_couchdb_admin_password@127.0.0.1:5984/gpii/_bulk_docs`.

Before I used the following command to interact with CouchDB from rake shell: `rake sh["kubectl exec --namespace gpii couchdb-couchdb-0 -c couchdb -- curl -s http://$TF_VAR_secret_couchdb_admin_username:$TF_VAR_secret_couchdb_admin_password@127.0.0.1:5984/gpii/_all_docs"]`

File: gcp/README.md [hunk 2]
```diff
@@ -630,16 +630,13 @@ Note that we assume that you are going to perform these steps into an already up
...
-1. Open a port forwarding between the cluster's couchdb host:port and your local machine: `kubectl --namespace gpii port-forward couchdb-0 5984`
-
-The port forwarding will be there until you hit _Ctrl-C_, so leave it running until we are done loading the preferences sets.
+1. Set up the env you are going to deal with: `rake sh`
 __Note__ that if you are going to perform this in production (prd) you should do it from the _prd_ folder and remember to use the _RAKE_REALLY_RUN_IN_PRD=true_ variable when issuing the commands against the production cluster.
+1. Open a port forwarding between the cluster's couchdb host:port and your local machine: `kubectl --namespace gpii port-forward couchdb-couchdb-0 5984 &`
+1. Get the current credentials for connecting to the couchdb from a flowmanager pod: `eval $(kubectl -n gpii exec flowmanager-[POD_ID] env | grep ^GPII_DATASOURCE_HOSTNAME | cut -f1 -d@)`
```
Comment (natarajaya) [inline gcp/README.md]: Please check my previous comment.

**Comment by mrtyler** [comment]
These are fine cleanups, but this is not what I requested. https://github.com/GPII/universal/pull/769#discussion_r271407954

**Comment by amatas** [comment]
@mrtyler I added the procedure I followed in the last 2 loads. 
The JSON files were always generated by Javi and then he sent them to me for loading them. 
Are you requesting the guide about how to build the JSON files?

File: gcp/README.md (hunk 1 above)
Comment (amatas) [reply gcp/README.md]: done, thansk @natarajaya

File: gcp/README.md (hunk 2 above)
Comment (amatas) [reply gcp/README.md]: done!

**Comment by mrtyler** [comment]
> Are you requesting the guide about how to build the JSON files?

Yes. The goal of this task was to add documentation we will need to start the system from scratch -- https://issues.gpii.net/browse/GPII-2806.

**Comment by amatas** [comment]
As we decided yesterday I'm going to merge this PR, the task is not completed as we still need the instructions about how to initialize the DB in the case of a disaster.

**Pull request merged**